### PR TITLE
Fix backup Url, export env variables to crontab

### PR DESCRIPTION
**Pull request by ozairbilal**

### CODE REVIEW

File: mysql-backup-restore-s3/run.sh [hunk 1]
```diff
@@ -102,7 +103,7 @@ elif [ -n "${INIT_RESTORE_LATEST}" ]; then
     ls -d -1 /backup/* | tail -1 | xargs /restore.sh
 fi
 
-echo "${CRON_TIME} /backup.sh >> /mysql_backup.log 2>&1" > /crontab.conf
+echo "${CRON_TIME} export S3_BUCKET_NAME=${S3_BUCKET_NAME}; export AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}; export AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}; export AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}; /backup.sh >> /mysql_backup.log 2>&1" > /crontab.conf
```
Comment (rasheedamir) [inline mysql-backup-restore-s3/run.sh]: @ozairaurora plz remove this echo

Comment (rasheedamir) [inline mysql-backup-restore-s3/run.sh]: @ozairaurora why everything on same line?

File: mysql-backup-restore-s3/run.sh [hunk 2]
```diff
@@ -77,13 +77,28 @@ echo "=> Creating restore script"
 rm -f /restore.sh
 cat <<EOF >> /restore.sh
```
Comment (hazim1093) [inline mysql-backup-restore-s3/run.sh:78]: @rasheedamir , @ozairaurora 
why are we creating a separate file restore.sh inside run.sh everytime, and execute in from within the same script everytime?

Comment (ozairbilal) [inline mysql-backup-restore-s3/run.sh:78]: In future if we are planning to trigger restore explicitly, this would help.

**Pull request merged**